### PR TITLE
[IMP] l10n_lu: taxes

### PR DESCRIPTION
**Pull request by malb-odoo**
We want to invigorate a Tax Name Taxonomy so that tax names are as codified as possible. This allow for a better display on Invoice Description, and allowed us to implement a smart name_search.

In this PR, we have changed the Taxes names so that it's more clear for users

task-id: 3052677

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_lu/data/template/account.tax-lu.csv [hunk 1]
```diff
@@ -1,1007 +1,1007 @@
 "id","sequence","description","invoice_label","name","amount","amount_type","type_tax_use","tax_group_id","active","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/factor_percent","repartition_line_ids/account_id"
-"lu_2011_tax_AB-EC-0","171","","0%","EX-EC-P-G","0.0","percent","purchase","tax_group_0","","base","invoice","+195","",""
+"lu_2011_tax_AB-EC-0","171","","0%",0% EX EC G,"0.0","percent","purchase","tax_group_0","","base","invoice","+195","",""
```
Comment (epictete) [inline addons/l10n_lu/data/template/account.tax-lu.csv:2]:
```suggestion
"lu_2011_tax_AB-EC-0","171","","0%","0% EX EC G","0.0","percent","purchase","tax_group_0","","base","invoice","+195","",""

```
Several occurrences :wink:

**Review by epictete — COMMENTED** [review]
@robodoo r+

**Comment by epictete** [comment]
@robodoo r-
There is a duplicate in tax names that we want to fix.

**Comment by epictete** [comment]
=============== There are duplicates name with the same tax type ===============
0% S           : [960, 996]

File: addons/l10n_lu/data/template/account.tax-lu.csv [hunk 2]
```diff
@@ -1,1007 +1,1007 @@
...
 "","","","","","","","","","","tax","invoice","+706","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-705","",""
 "","","","","","","","","","","tax","refund","-706","","lu_2020_account_461411"
-"lu_2015_tax_SANS","506","","0%","0-P-Tax-Free","0.0","percent","purchase","tax_group_0","","base","invoice","","",""
+"lu_2015_tax_SANS","506",Tax free,"0%",0%,"0.0","percent","purchase","tax_group_0","","base","invoice","","",""
 "","","","","","","","","","","tax","invoice","","",""
 "","","","","","","","","","","base","refund","","",""
 "","","","","","","","","","","tax","refund","","",""
-"lu_2015_tax_SANS_sale","507","","0%","0-S-Tax-Free","0.0","percent","sale","tax_group_0","","base","invoice","","",""
+"lu_2015_tax_SANS_sale","507","","0%",0% S S,"0.0","percent","sale","tax_group_0","","base","invoice","","",""
```
Comment (epictete) [inline addons/l10n_lu/data/template/account.tax-lu.csv:996]:
```suggestion
"lu_2015_tax_SANS_sale","507","","0%","0% S S","0.0","percent","sale","tax_group_0","","base","invoice","","",""

```
:scream:

File: addons/l10n_lu/data/template/account.tax-lu.csv [hunk 3]
```diff
@@ -1,1007 +1,1007 @@
...
 "","","","","","","","","","","tax","invoice","+040","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-031","",""
 "","","","","","","","","","","tax","refund","-040","","lu_2020_account_461411"
-"lu_2015_tax_VP-PA-8","505","","8%","8-S-S","8.0","percent","sale","tax_group_8","","base","invoice","+705","",""
+"lu_2015_tax_VP-PA-8","505","","8%","8% S","8.0","percent","sale","tax_group_8","","base","invoice","+705","",""
 "","","","","","","","","","","tax","invoice","+706","","lu_2020_account_461411"
 "","","","","","","","","","","base","refund","-705","",""
 "","","","","","","","","","","tax","refund","-706","","lu_2020_account_461411"
-"lu_2015_tax_SANS","506","","0%","0-P-Tax-Free","0.0","percent","purchase","tax_group_0","","base","invoice","","",""
+"lu_2015_tax_SANS","506",Tax free,"0%",0%,"0.0","percent","purchase","tax_group_0","","base","invoice","","",""
```
Comment (epictete) [inline addons/l10n_lu/data/template/account.tax-lu.csv:992]:
```suggestion
"lu_2015_tax_SANS","506",Tax free,"0%","0%","0.0","percent","purchase","tax_group_0","","base","invoice","","",""

```
I hope there aren't any more I didn't see :fearful:

**Review by epictete — CHANGES_REQUESTED** [review]

File: addons/l10n_lu/data/template/account.tax-lu.csv (hunk 2 above)
Comment (malb-odoo) [reply addons/l10n_lu/data/template/account.tax-lu.csv:996]: I did'nt had the regex for this loca, so the others should be good :rofl:

**Comment by epictete** [comment]
@robodoo r+

**Pull request closed without merging**